scripts/build.py

```diff
@@ -146,6 +145,7 @@ def get_cache_slug():
     ASSETS_OUT_DIR = os.path.join(BUILD_DIR, 'assets', '')
     copy_files_source_to_target(ASSETS_SRC_DIR, ASSETS_OUT_DIR)
 
+    process_third_party_libs()
```

Comment:
I'm not sure I understand why we need to process and then copy. Could we get the build directory to just be THIRD_PARTY_GENERATED_OUT_DIR instead?